packages/manager/modules/pci/src/projects/project/components/activate-project-banner/activate-project-banner.component.js

```diff
@@ -0,0 +1,11 @@
+import template from './activate-project-banner.html';
+import controller from './activate-project-banner.controller';
+import './index.scss';
+
+export default {
+  bindings: {
+    callback: '&',
```

Comment:
Please rename to something more relevant
```suggestion
    onClick: '&',
```